packages/shared/components/inputs/DateTimePicker.svelte

```diff
@@ -43,8 +44,12 @@
         on:change={tooltip?.refreshPosition}
     />
     <div class="flex flex-row justify-center items-center space-x-4 w-full">
-        <Button small secondary onClick={handleCancelClick} classes="w-full">{localize('actions.cancel')}</Button>
-        <Button small onClick={handleConfirmClick} classes="w-full">{localize('actions.confirm')}</Button>
+        <Button size={ButtonSize.Small} outline onClick={handleCancelClick} classes="w-full"
```

Comment:
To me these buttons look like they are missing some padding 
![image](https://user-images.githubusercontent.com/3624944/190412755-5daf6b67-9d00-4dca-966c-aed9eb2e2a3c.png)